docker/config/local_dev.env

```diff
@@ -70,15 +70,15 @@ resource.boto.resource_class=socorro.external.boto.connection_context.HostPortS3
 # there's only one processor node. For server environments, we probably want to store that
 # in a volume. These three vars are all affected.
 companion_process.symbol_cache_path=/tmp/symbols/cache
-processor.symbol_cache_path=/tmp/symbols/cache
-processor.symbol_tmp_path=/tmp/symbols/tmp
+processor.breakpad.symbol_cache_path=/tmp/symbols/cache
+processor.breakpad.symbol_tmp_path=/tmp/symbols/tmp
```

Comment:
These variables are now namespaced so that `BreakpadStackwalkerRule2015` configuration is separate from `JitCrashCategorizeRule` configuration.